account_multicurrency_revaluation/demo/account_demo.xml

```diff
@@ -1,66 +1,105 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="res_partner_reval" model="res.partner">
+        <field name="name">Revaluation Company</field>
+        <field eval="1" name="customer"/>
+        <field eval="1" name="supplier"/>
+        <field name="is_company">1</field>
```

Comment:
Just for my information: I guess you could have used
```
<field eval="1" name="is_company"/>
```
here, as for the lines around, correct? If yes, what's the recommended way?

Reply:
IMHO this is two method how to do the same things.
in this case, it is better to follow one style.